include/singa/neuralnet/neuron_layer.h

```diff
@@ -122,13 +122,17 @@ class DropoutLayer : public NeuronLayer {
  */
 class DummyLayer: public NeuronLayer {
  public:
+  void Setup(const std::string str, const vector<Layer*>& srclayers);
   void Setup(const LayerProto& proto, const vector<Layer*>& srclayers) override;
   void ComputeFeature(int flag, const vector<Layer*>& srclayers) override;
   void ComputeGradient(int flag, const vector<Layer*>& srclayers) override;
+  void Feed(int batchsize, vector<float>& data, int is_aux);
```

Comment:
I mean the following signature instead of renaming the argument 'op->is_aux'.

Feed(const vector<int>& shape, const vector<float*>& data, const vector<int>& aux_data);